eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Deleg.hs

```diff
@@ -79,14 +82,18 @@ import NoThunks.Class (NoThunks (..))
 data DelegEnv era = DelegEnv
   { slotNo :: SlotNo
   , ptr_ :: Ptr
-  , acnt_ :: AccountState
+  , account_ :: AccountState
```

Comment:
Can you revert this rename?

We should address this type separately, cause all fields in this type are named terribly and `account_` is not much better of a name :cry: